docs/client-sdk/javascript/tutorials/quickstart.md

```diff
@@ -175,6 +175,60 @@ for await (const conversation of stream) {
 }
 ```
 
+#### Hide empty conversations
+
+In some edge cases, apps might unintentionally create empty conversations. Displaying empty conversations in your app can create a confusing UX. For this reason, hide empty conversations.
```

Comment:
Would it be helpful to say why empty conversations are created sometimes?

May I have some help with the code sample below, please? This is from ChatGPT!